apps/portal/src/app/database.py

```
@@ -0,0 +1,194 @@
+
+def main():
+    conn_params = {
+        'dbname': os.getenv('DB_NAME', 'project_database'),
+        'user': os.getenv('DB_USER', 'postgres'),
+        'password': os.getenv('DB_PASSWORD', '1234'),
+        'host': os.getenv('DB_HOST', 'localhost'),
+        'port': os.getenv('DB_PORT', '5432')
+    }
+
+    with psycopg.connect(**conn_params) as conn:
+        with conn.cursor() as cur:
```

Ruff

Use a single `with` statement with multiple contexts instead of nested `with` statements

Combine `with` statements

(SIM117)